pkg/codec.go

```diff
@@ -443,3 +443,23 @@ func readBody(header *dubbo_spec.DubboHeader, in remote.ByteBuffer) ([]byte, err
 	}
 	return in.Next(length)
 }
+
+func getJavaClassName(message remote.Message) string {
```

Comment:
Better provide a more comprehensive warning to instruct the user to add the annotation in his IDL.

Reply:
Ack.